R/source.R

```diff
@@ -215,6 +207,49 @@ rust_function <- function(code, env = parent.frame(), ...) {
   rust_source(code = code, env = env, ...)
 }
 
+invoke_cargo <- function(
+  toolchain, specific_target,
+  dir, profile, stdout, stderr,
+  use_rtools) {
```

Comment:
Function arguments are formatted a bit strangely here. We normally follow the format you see e.g. in `generate_cargo.toml()` on lines 253/254.